apparmor.d/groups/lxqt/lxqt-globalkeysd

```diff
@@ -0,0 +1,42 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-globalkeysd
+profile lxqt-globalkeysd @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/bus-accessibility>
+  include <abstractions/video>
+  include <abstractions/lxqt>
+  include <abstractions/nameservice-strict>
+  include <abstractions/qt5-shader-cache>
+  include <abstractions/gvfs-open>
```

Comment:
Don't use `gvfs-open` directly. You need someting like: `@{open_path} rPx -> child-open,` here.

Reply:
ok, its used in lxqt-globalkeysd. I altered it in 2 other occurences in profiles to be considered later